org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Replay/ImportReplay.java

```diff
@@ -1242,6 +1271,66 @@ final private String getThisSymProperty(final String symbology,
     }
     return null;
   }
+  
+  private List<Editable> addElementsToExistingLayer(Layer layerToAddTo,Editable l)
+  {
+    List<Editable> elementsAdded = new ArrayList<>();
+    Enumeration<Editable> tempElements = ((Layer)l).elements();
+    while(tempElements.hasMoreElements()) {
+      Editable elem = tempElements.nextElement();
+      layerToAddTo.add(elem);
+      elementsAdded.add(elem);
+    }
+    return elementsAdded;
+  }
+  
+  public final void importThis(final String text) {
```

Comment:
there's already an `importThis` method in this class.  It does lots of other _tidying up_ steps while doing the import.  

I _think_ it would be safer for us to re-use that method, since users will _probably_ expect that tidying up to happen, I think we should re-use that method.

To do that, I think we would extract the code to that puts the InputStream into a list of lines, then make a call to a common method that does all the other processing. Clearly this second method is the one that our PasteRepAction will call.

_Tip: since this is refactoring, we'll help ourselves by doing it after the new/updated unit testing is implemented. These tests will give us confidence that our refactoring is safe_.